Extensions/AnchorBehavior/anchorruntimebehavior.ts

```diff
@@ -110,13 +110,15 @@ namespace gdjs {
         }
 
         //Calculate the distances from the window's bounds.
-        const topLeftPixel = this._convertCoords(
-          instanceContainer,
-          layer,
-          this.owner.getDrawableX(),
-          this.owner.getDrawableY(),
-          workingPoint
-        );
+        const topLeftPixel = this._relativeToOriginalWindowSize
+          ? [this.owner.getDrawableX(), this.owner.getDrawableY()]
+          : this._convertInverseCoords(
+              instanceContainer,
+              layer,
+              this.owner.getDrawableX(),
+              this.owner.getDrawableY(),
+              workingPoint
+            );
```

Comment:
Should this be inside `_convertInverseCoords` next to `isParentACustomObject`?
Maybe `isParentACustomObject` is no longer needed with your fix?

Reply:
Let's assume for now it's still useful (and yeah the issue was the wrong base was used, so at least it's better now)